trin-core/src/portalnet/overlay_service.rs

```diff
@@ -79,6 +79,20 @@ const EXPECTED_NON_EMPTY_BUCKETS: usize = 17;
 /// Bucket refresh lookup interval in seconds
 const BUCKET_REFRESH_INTERVAL_SECS: u64 = 60;
 
+/// An action that the overlay may perform.
+#[derive(Debug)]
```

Comment:
I think it is a good practice to implement also `Clone` and `PartialEq/Eq` traits.

Reply:
I don't think those traits apply here. You cannot clone `oneshot::Sender`, and I cannot think of any circumstances where we would compare two commands.